redhat/lyrionmusicserver.spec

```diff
@@ -392,12 +351,12 @@ function setSYSV {
         test -f /tmp/squeezerpmdebug && set -x
 
 	# This is a SYSV server. Copy SYSV script to the correct place.
-	cp -p %{_datadir}/%{shortname}/%{shortname}.SYSV %{_sysconfdir}/init.d/%{shortname} >/dev/null 2>&1 || :
+	/usr/bin/cp -p %{_datadir}/%{shortname}/%{shortname}.SYSV %{_sysconfdir}/init.d/%{shortname} >/dev/null 2>&1 || :
 
-	#Koozali SME Server pre version 10 uses SYSV init and uses runlevel 7
+	# Koozali SME Server pre version 10 uses SYSV init and uses runlevel 7
```

Comment:
Wow... Koozali! I used to use e-smith (Koozali's grandfather) quite a bit many years ago 😀.

Reply:
OK, fine will fix that